nuntium/templates/nuntium/profiles/welcome.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base_manager.html" %}
+{% load i18n %}
+
+{% block header %}
+{% include 'nuntium/profiles/top_menu.html' with section='your-instances' %}
```

Comment:
This is highlighting the wrong tab - I think you probably need to remove the `with section=`.